playbook/app/pb_kits/playbook/pb_radio/radio.html.erb

```diff
@@ -1,18 +1,42 @@
-<%= content_tag(:label,
-    aria: object.aria,
-    checked: object.checked,
-    class: object.classname,
-    data: object.data,
-    id: object.id,
-    value: object.value,
-    **combined_html_options) do %>
-
-    <% if content.present? %>
-      <%= content %>
-    <% else %>
-      <%= radio_button_tag object.name, object.value, object.selected, object.input_options %>
+<% if object.custom_children %>
+  <div data-pb-radio-children="true">
+    <%= pb_rails("flex", props: {
+      aria: object.aria,
+      align: "center",
+      class: object.classname,
+      cursor: "pointer",
+      **combined_html_options
+    }) do %>
+      <%= content_tag(:label,
+        checked: object.checked,
+        class: object.classname,
+        data: object.data,
+        id: object.id,
+        value: object.value,
+        **combined_html_options) do %>
```

Comment:
Do we want to keep the data, aria and html_options props on the label and not the top level div? With this set up applying global props to this kit is applying the global prop only to the input, not the entire kit which is not what we want. We want global props to apply to the top level div for the kit.
![Screenshot 2024-09-16 at 11 57 09 AM](https://github.com/user-attachments/assets/fec9341c-4960-441b-b944-e0c6695f30eb)